docs/source/operations/options/scale.rst

```diff
@@ -0,0 +1,5 @@
+.. option:: +scale=<value>
+
+    Ellipsoid scale factor. Determines ellipsoid scale factor used in the projection.
```

Comment:
As this is a very specific options to this projection method, I would keep it inline in lccm.rst

Reply:
Yes, it makes sense, will change that.

I was also not sure about the name 'scale'. I think 'k_0' (and 'k' in the past) is generally used for scaling, but since this is a different kind of scaling I decided that it would be better to have a separate option than to reuse 'k_0' somehow ('k_0' can still be used in conjunction with 'scale' though).